paper/sections/1intro.tex

```diff
@@ -51,7 +50,7 @@ \section{Introduction}
 Our results show that VRF-based mining is easy to implement and introduces little overhead.
 
 \textbf{Partial outsourcing is unrealistic.}
-Partial outsourcing is that, a pool operator interactively outsources computation that does not need the secret key to miners.
+Partial outsourcing is that, a pool operator interactively outsources computation to miners, without suffering from the loss of funds or rewards.
```

Comment:
For this part, I prefer the original version. The key issue of the pool is that he may lose sk by outsourcing.